### DISCUSSION TITLE
Plans to support Maven Repository Access without Personal Access Token?

### DISCUSSION THREAD

**Discussion by OliverMeyerDSA** [community/community#130728] [Packages]
### Select Topic Area

Product Feedback

### Body

Hi

To retrieve artifacts from a maven repository of a private repository you need a personal access token (classic). That is not a good choice from a team/company security point of view, because I have to enable PAT access globally. 

Are there any plans to change that? I really like to be able to use organization based fine grained access tokens or the GITHUB_TOKEN to access private maven registries from the same company.

I am currently considering to create another private repository outside the company just to store maven deployments. Then I could still forbid PAT access to the company repositories. Am I wrong in considering that an avoidable work around?

Thanks
Oliver

**Comment by ZsoltMolnarrr** [comment]
Hi!

I would also greatly appreciate this feature.

**Comment by albrektsson** [comment]
+1 

Looks like there was a plan but it was moved back to the backlog this summer: https://github.com/orgs/github/projects/4247/views/1?filterQuery=is%3Aopen+label%3Apackages
